test_examples/module_emr-iam/main.tf

```
@@ -0,0 +1,13 @@
+module "emr-iam" {
+  source = "../../modules/aws-emr-iam"
+
+  s3_bucket_name_for_logs           = var.s3_bucket_name_for_logs
+  s3_bucket_name_for_root_directory = var.s3_bucket_name_for_root_directory
+  s3_policy_arns                    = var.s3_policy_arns
```

This was recently changed to `additional_policy_arns`
https://github.com/Datatamer/terraform-aws-emr/tree/master/modules/aws-emr-iam
```suggestion
  additional_policy_arns            = var.s3_policy_arns
```